pkg/backend/display/progress.go

```diff
@@ -1027,6 +1027,23 @@ func (display *ProgressDisplay) printSummary() {
 
 	msg := renderSummaryEvent(*display.summaryEventPayload, false, display.opts)
 	display.println(msg)
+
+	// after printing resources changes summary also add the count for resources that errored
+	resourcesErrored := 0
+
+	rr := toResourceRows(display.eventUrnToResourceRow, display.opts.DeterministicOutput)
+
+	for _, r := range rr {
+		if r.DiagInfo().ErrorCount > 0 {
+			resourcesErrored++
+		}
+	}
+
+	if resourcesErrored > 0 {
+		errSummaryStr := fmt.Sprintf("%d errored", resourcesErrored)
+		display.println("    " + colors.Red + errSummaryStr + colors.Reset)
+		display.println("")
```

Comment:
I think this needs to be moved out of the `if`, to replace the line you removed here: https://github.com/pulumi/pulumi/pull/20534/files#diff-4d8b36285a68d688efdec3d43b7d4f77fb0f345e8075fb509fd6a5a56d66626e